pkg/innerring/innerring.go

```diff
@@ -63,6 +64,8 @@ type (
 	Server struct {
 		log *logger.Logger
 
+		blockChain *blockchain.Blockchain
```

Comment:
s/blockChain/blockchain?

Reply:
package name collision, but here legit

Reply:
Make it just ‘bc’ like everywhere in neo-go?